x-pack/plugins/security_solution/public/flyout/document_details/right/tabs/table_tab.tsx

```diff
@@ -45,7 +46,12 @@ export const getColumns: ColumnsProvider = ({
     width: '30%',
     render: (field, data) => {
       return (
-        <FieldNameCell data={data as EventFieldsData} field={field} fieldMapping={undefined} />
+        <FieldNameCell
```

Comment:
Any chance you can add a Unit test for this?

Reply:
Absolutely